packages/editor/src/plugins/image/controls/upload-button.tsx

```diff
@@ -1,27 +1,42 @@
 import { EditorTooltip } from '@editor/editor-ui/editor-tooltip'
 import { TempFile, isTempFile } from '@editor/plugin'
-import { faCircleArrowUp, faRedoAlt } from '@fortawesome/free-solid-svg-icons'
+import { faRedoAlt, faUpload } from '@fortawesome/free-solid-svg-icons'
 import { FaIcon } from '@serlo/frontend/src/components/fa-icon'
 import { useEditorStrings } from '@serlo/frontend/src/contexts/logged-in-data-context'
-import { cn } from '@serlo/frontend/src/helper/cn'
+import { useState } from 'react'
 
 import type { ImageProps } from '..'
+import { cn } from '@/helper/cn'
 
 export function UploadButton({ config, state }: ImageProps) {
   const { src } = state
   const imageStrings = useEditorStrings().plugins.image
   const isFailed = isTempFile(src.value) && src.value.failed
 
+  const [isLabelFocused, setIsLabelFocused] = useState(false)
   return (
     <>
       <label
-        className={cn(`
-          mr-2 cursor-pointer rounded-md border border-gray-500
-          px-1 text-sm transition-all focus-within:bg-editor-primary-200 hover:bg-editor-primary-200
-          focus-visible:bg-editor-primary-200
-        `)}
+        onFocus={(e) => {
+          e.stopPropagation()
+          setIsLabelFocused(true)
+        }}
+        onBlur={(e) => {
+          e.stopPropagation()
+          setIsLabelFocused(false)
+        }}
+        className={cn(
+          'mb-4 flex w-full py-2',
+          'items-center justify-center',
+          'cursor-pointer rounded-lg',
+          'font-semibold text-gray-800',
+          'bg-editor-primary-200 hover:bg-editor-primary-300',
+          isLabelFocused ? 'outline outline-2 outline-red-400' : ''
```

Comment:
Should it really be red, if the other elements are blue when focused?

Reply:
All button elements in this screen have red outlines when they're focused in my browser, are they blue for you?

Reply:
![image](https://github.com/serlo/frontend/assets/161819227/bcb93069-f805-4c64-82d7-5aebe4b757f8)

Reply:
Yeah, all were blue for me, except for the upload one.

<img width="818" alt="Screenshot 2024-06-27 at 12 17 22" src="https://github.com/serlo/frontend/assets/13198821/c436c421-0282-4195-a441-7b96fe406ba0">

Reply:
The red border around the text field should appear **ONLY** if URL is invalid/incomplete

Reply:
And on focus, no?

Reply:
Yep

Reply:
Done :heavy_check_mark: 

Need to investigate why I see red on my screen